cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt

```diff
@@ -6,13 +6,18 @@ import java.util.*
 
 private const val HEADLESS_FLAG = "--headless"
 private const val CAPSULE_DEBUG_FLAG = "--capsule-debug"
+private const val BASE_DEBUG_PORT_FLAG = "-Dnet.corda.noderunner.debug.port="
+private const val BASE_MONITORING_PORT_FLAG = "-Dnet.corda.noderunner.monitoring.port="
 private const val CORDA_JAR_NAME = "corda.jar"
```

Comment:
These are no longer Java system properties and so we can use flags values like
```
--debug-port
--monitoring-port
```
instead.